assets/js/translator.js

```diff
@@ -4,10 +4,39 @@ var curSrcLang, curDstLang;
 var recentSrcLangs = [], recentDstLangs = [];
 var droppedFile;
 var textTranslateRequest;
+var recaptchaScriptSrc = "https://www.google.com/recaptcha/api.js?onload=recaptchaRenderCallback&render=explicit&hl=";
+
+var recaptchaRenderCallback = function () {
```

Comment:
Can we move this guy after L15? Leave a line after it before what's current L16. We try to minimize our clobbering of the global namespace.